### PR TITLE
fix centos7 test

### LINKED ISSUE #16: Molecule test for centos7 fails
The firewalld service is failing to start for centos7 molecule test.

### PR DESCRIPTION
**Pull request by drmatthews**
Changes:

- Use new molecule composite action
- Address centos7 molecule failure

Fixes #16

### CODE REVIEW

File: molecule/resources/converge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  #     register: systemctl_status
+  #     until: >
+  #       'running' in systemctl_status.stdout or
+  #       'degraded' in systemctl_status.stdout
+  #     retries: 30
+  #     delay: 5
+  #     when: ansible_service_mgr == 'systemd'
+  #     changed_when: false
+  #     failed_when: systemctl_status.rc > 0
+
```
Comment (p-j-smith) [inline molecule/resources/converge.yml:17]: can these lines be removed?

**Review by p-j-smith — CHANGES_REQUESTED** [review]

**Comment by p-j-smith** [comment]
nice! do you know why it was failing before?

**Comment by drmatthews** [comment]
I do but I can't explain it! All the guidance about molecule and systemd says that you need:

```yaml
volumes:
   - /sys/fs/cgroup:/sys/fs/cgroup:rw
```

But this just doesn't work for the `centos:7` Docker image.

Before you made the composite action I was trying the approach of having a single molecule scenario and passing in the Docker image to use. I did this because so I could use the same Dockerfile and molecule config for each image. Couldn't get it to work and needed separate scenarios for the composite action.

File: molecule/resources/converge.yml (hunk 1 above)
Comment (drmatthews) [reply molecule/resources/converge.yml]: Thanks for spotting that!

**Review by p-j-smith — APPROVED** [review]
looks good!

**Pull request merged**